static_src/util/format_date.js

```diff
@@ -1,18 +1,17 @@
 import moment from 'moment-timezone';
 
-const invalidMsg = 'Invalid datetimes cannot be formatted.';
+const makeInvalidDateTimeValueError = str =>
+  new Error(`could not format invalid date/time value: ${str}`);
 
-/**
```

Comment:
no need to remove valid comments

Reply:
I can revert. One of the reason these kinds of comments are not great is that they become outdated unless updated when the function is updated. When I went to update the comment I realised I was describing what the ~4 line function was doing, so it seemed redundant, so I removed it.